defaults/main.yml

```diff
@@ -10,3 +10,26 @@ geoip2_maxmind_editions:
   - GeoIP2-City
 geoip2_maxmind_suffix_file: tar.gz
 geoip2_maxmind_suffix_checksum: tar.gz.sha256
+
+# Indicate if MaxMind's geoipupdate tool (https://github.com/maxmind/geoipupdate)
+# should be installed and which version to install. Note that the version must
+# represent a release in the maxmind/geoipupdate GitHub repository.
+geoip2_install_geoipupdate: false
+# We do not want this to be treated as a numeric value so we quote it to ensure
+# it is treated as a string.
+geoip2_geoipupdate_version: "7.0.1"
+
+# Indicate if a SystemD service and timer should be added to automatically
+# update the GeoIP database files when the geoipupdate tool is installed.
+geoip2_geoipupdate_auto_update: true
+# The name to use for the SystemD service and timer that automatically updates
+# the GeoIP database files.
+geoip2_geoipupdate_service_name: geoipupdate
+# Configure the timer that runs the geoipupdate service. The database update
+# schedule at
+# https://support.maxmind.com/hc/en-us/articles/4408216129947-Download-and-Update-Databases#h_01G3XX402XKD3J1CMWKNKMDYYZ
+# indicates that most databases update on Tuesday and Friday as late as 23:59 EST.
+# Therefore, the default is to update the databases on Wednesdays and Saturdays
+# between 00:00 and 03:00 EST.
+geoip2_geoipupdate_service_timer_on_calendar: Wed,Sat America/New_York
+geoip2_geoipupdate_service_timer_randomized_delay_sec: 3h
```

Comment:
Same confusion here as above re: `3h` vs. seconds.

Reply:
Please let me know if [9c1a1eb](https://github.com/cisagov/ansible-role-geoip2/pull/51/commits/9c1a1eb0d15c22bfd6ee139b38d48dc7d715efdf) clarifies things.

Reply:
I'm in agreement with @jsf9k's suggestion [here](https://github.com/cisagov/ansible-role-geoip2/pull/51/files#r1664226422).

Reply:
Please see [450b394](https://github.com/cisagov/ansible-role-geoip2/pull/51/commits/450b394807a674789207cfc2951c2d16a59c19e1). I do not want to burn time trying to document the peculiarities of systemd's functionality, so I would rather just reference the options being populated and leave it to the reader.

Reply:
I'm on board with the updated variable description, but I think this comment should remain where you had it in `defaults/main.yml`:
```
# This value must be a valid systemd.time time span value. Please see the
# documentation for more information:
# https://www.freedesktop.org/software/systemd/man/latest/systemd.time.html
```

Reply:
Ok. I only removed it because it was my own interpretation after researching. The [option's documentation](https://www.freedesktop.org/software/systemd/man/latest/systemd.timer.html#RandomizedDelaySec=) is not super helpful. Please see [3d9648a](https://github.com/cisagov/ansible-role-geoip2/pull/51/commits/3d9648aef4e4630e07cca31cb320913fafb566f7).

Reply:
@dav3r Any other concerns on this suggestion?

Reply:
Nope, all good - resolving now!